resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -524,7 +533,20 @@ private InetSocketAddress nextNameServerAddress() {
             InetSocketAddress nameServerAddr, DnsQuestion question) {
 
         return query0(checkNotNull(nameServerAddr, "nameServerAddr"),
+                checkNotNull(question, "question"),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "question" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1192)